templates/20-04/azure.html

```diff
@@ -0,0 +1,227 @@
+{% extends "20-04/base_20-04.html" %}
+
+{% from "_macros/vf_hero.jinja" import vf_hero %}
+{% from "_macros/vf_tiered-list.jinja" import vf_tiered_list %}
+{% from "_macros/vf_quote-wrapper.jinja" import vf_quote_wrapper %}
+
+{% block title %}Ubuntu 20.04 LTS (Focal Fossa) EOS on Azure{% endblock %}
+
+{% block meta_description %}
+  Ubuntu 20.04 LTS goes out of support in May. Upgrade to Ubuntu Pro or move to a newer version of Ubuntu to maintain a supported OS. Learn more.
+{% endblock %}
+
+{% block meta_keywords %}Ubuntu on Azure, Secure VMs{% endblock %}
+
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1qlz7QbmUGHouIvq95XuJbQkhzitzG7UupBKq_Bl2zfg/edit
+{% endblock meta_copydoc %}
+
+{% block content %}
+
+  {% call(slot) vf_hero(
+    title_text='Ubuntu 20.04 LTS <br class="u-hide--small" />(Focal Fossa) on Azure',
+    layout='50/50'
+    ) -%}
+    {%- if slot == 'description' -%}
+      <p>
+        Ubuntu 20.04 LTS (Focal Fossa) is reaching the end of its 5 years of standard support on May 31, 2025, exposing your systems to unpatched vulnerabilities. To stay secure, you must act now. Explore your options below to ensure continued protection.
+      </p>
+    {%- endif -%}
+    {%- if slot == 'cta' -%}
+      <a href="https://documentation.ubuntu.com/azure/en/latest/azure-how-to/instances/get-ubuntu-pro/"
+         class="p-button--positive">Upgrade on Azure</a>
+      <a href="https://www.brighttalk.com/webcast/6793/638246"
+         class="p-button">Watch our webinar</a>
+    {%- endif -%}
+    {%- if slot == 'image' -%}
+      <div class="p-image-container--3-2 is-cover is-highlighted u-hide--small u-hide--medium">
+        {{ image(url="https://assets.ubuntu.com/v1/74c5e0ac-hero-img.png",
+                alt="",
+                width="1200",
+                height="800",
+                hi_def=True,
+                loading="auto") | safe
+        }}
+      </div>
+    {%- endif -%}
+  {% endcall -%}
+
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
+        <h2>Standard support for Ubuntu 20.04 LTS  ends on May 31, 2025</h2>
+      </div>
+      <div class="col">
+        <p>
+          Transitioning to an Ubuntu version with active standard support (Ubuntu 22.04 LTS or 24.04 LTS) is crucial for performance, hardware compatibility, and access to new technologies, making it ideal for new workloads. However, for existing deployments, the process may be more complex.
+        </p>
+        <div class="p-cta-block">
+          <a href="https://ubuntu.com/about/release-cycle">Learn more about the Ubuntu lifecycle and release cadence&nbsp;&rsaquo;</a>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="u-fixed-width">
+      <hr class="p-rule" />
+      <div class="p-section--shallow">
+        <h2>What does this mean for you?</h2>
```

Comment:
Where does this title come from? Both in the copy doc and in the design it shows "What are your options?"

Reply:
Hm… Good catch. Not sure what happened there.